src/components/Books/Books.js

```diff
@@ -1,27 +1,32 @@
-import React, { useEffect, useState } from "react";
+import React, { useEffect } from "react";
 import { useSelector, useDispatch } from "react-redux";
 import { LoadBooks } from "./BooksActions";
 import Book from "../Book/Book";
 
 import { CardDeck, Spinner } from "react-bootstrap";
+import { SetMaximumPages } from "../Pagination/PaginationActions";
+import Pagination from "../Pagination/Pagination";
+import PrivateRoute from "../PrivateRoute/PrivateRoute";
 
-const Books = props => {
+const Books = () => {
   const books = useSelector(state => state.BooksReducer);
```

Comment:
Since you are pulling two values from state here you can use destructuring.

Example:
```
const { books, pages } = useSelector(state => ({ books, pages });
```